### ISSUE TITLE
Some more deinflections not working

### ISSUE THREAD

**Issue by timrae** [mvysny/aedict#525]
伝わり ([renyoukei form](http://www.epochrypha.com/japanese/materials/verbs/Verb.aspx?rule=renyoukei))
使われている (mix of passive and continative)

I'm trying to analyze some big sentences right now, so a few more are likely to come up

**Comment by mvysny** [comment]
Yes, the old deinflection engine is apparently reaching its limits. I am working on a new engine, but it will take some time to get it right. I will bend the old engine where possible, but e.g. 使われている would be a toughie. Please give me some time (cca 1 month) to finish the new engine.

**Comment by mvysny** [comment]
Okay, more than a year has passed and the new engine is not even started - sorry for that ;)
